packaging/linux/fedora/sunshine.spec

```diff
@@ -0,0 +1,237 @@
+%global build_timestamp %(date +"%Y%m%d")
+%undefine _hardened_build
+
+Name:           sunshine
+Version:        2024.911.215654  # TODO: can we get this dynamically?
```

Comment:
should be able to import this from a git tag https://docs.pagure.org/copr.copr/user_documentation.html#github-webhooks https://docs.pagure.org/copr.copr/user_documentation/pagure_integration.html#auto-rebuilding 

and use something like
`{{{ git_dir_version }}}`